bump_release_version_url.sh

```diff
@@ -1,9 +1,9 @@
 #!/bin/bash
 
 OLD_RELEASE="17\.03"
-NEW_RELEASE="18\.04"
+NEW_RELEASE="18\.05"
 
-for i in content/cloud.md content/desktop.md content/server.md; do
+for i in content/download/flavours/cloud/*.md content/download/flavours/desktop/*.md content/download/flavours/server/*.md; do
```

Comment:
Nice idea! I need to think about whether we could achieve it with [.Site.Params](https://gohugo.io/variables/site/#the-site-params-variable) in the future.